src/stylesheets/elements/_typography.scss

```diff
@@ -33,6 +33,27 @@ a {
   }
 }
 
+// External link consider "effortless style approach":
+// [href^="http:"]:not([href*="my-domain.com"])
+// [href^="https:"]:not([href*="my-domain.com"])
+
+.usa-external_link {
+  &::after {
+    display: inline-block;
+    width: $base-font-size;
+    height: $base-font-size;
+    margin-bottom: -1px;
+    margin-left: 4px;
+    content: '';
+    background: url('../img/external-link.svg') no-repeat 0 0;
```

Comment:
I thought most of our svg images had png fallbacks. Why is that not the case here?